### PR TITLE
mavlink-router: Use configuration files to set up endpoints

### PR DESCRIPTION
**Pull request by edersondisouza**
mavlink-routerd.sh script contains no more information about endpoints:
mavlink-router will load them from /etc/mavlink-router/main.conf and of
any conf file on /etc/mavlink-router/config.d dir.

This allows for one to simply override default configurations by placing
a new conf file on /etc/mavlink-router/config.d dir.

### CODE REVIEW

**Comment by zehortigoza** [comment]
Looks good to me!
But would be nice also update the baudrate on /usr/sbin/aerofc_update.sh
and check if it is using the new px_uploader.py that have support for multiple baudrates and use: --baud-flightstack 1500000,460800
As it will support users that have PX4 firmware running in both baudrates.

**Comment by edersondisouza** [comment]
@zehortigoza I didn't get 

> https://github.com/intel-aero/meta-intel-aero/pull/28

Do you mean a runtime check, or just check if `px_uploader.py` already supports multiple bauds?

**Comment by zehortigoza** [comment]
> Do you mean a runtime check, or just check if px_uploader.py already supports multiple bauds?
Check if the version of px_uploader.py on image supports if not we need the version that support and update the aerofc_update.sh script to use both baudrates.

**Comment by lucasdemarchi** [comment]
@edersondisouza just copy the current px_uploader.py (https://github.com/PX4/Firmware/blob/master/Tools/px_uploader.py) to the right place (recipes-support/scripts/files/px_uploader.py).

**Comment by lucasdemarchi** [comment]
... and update the aerofc_update.sh of course.

File: recipes-support/mavlink-router/files/main.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[General]
+tcp-server-port=5760
+report-stats=false
+
+[UartEndpoint uart]
+Device = /dev/ttyS1
+Baud = 460800
+
+[UdpEndpoint wifi-ap]
```
Comment (lucasdemarchi) [inline recipes-support/mavlink-router/files/main.conf]: I would name this "wifi" only. So people switching to client mode can simply drop a file in conf.d to update the address. Would be nice not to need the address, and simply pass the iface to be used, making mavlink-router to broadcast on all ip broadcast addresses of that interface. But that is something that needs to be implemented on mavlink-router

File: recipes-support/mavlink-router/files/main.conf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[General]
+tcp-server-port=5760
```
Comment (lucasdemarchi) [inline recipes-support/mavlink-router/files/main.conf]: TcpServerPort

File: recipes-support/mavlink-router/files/main.conf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+report-stats=false
```
Comment (lucasdemarchi) [inline recipes-support/mavlink-router/files/main.conf]: ReportStats

**Comment by lucasdemarchi** [comment]
For aerofc_update.sh, look here what we currently do on PX4/Firmware:
https://github.com/PX4/Firmware/blob/master/Tools/aero_upload.sh#L49-L71

This is basically what aerofc_update.sh should become... with the differences that it's being executed locally.

**Comment by edersondisouza** [comment]
Updated to address comments. Note that config file keys renaming depends on https://github.com/01org/mavlink-router/pull/44

**Comment by edersondisouza** [comment]
Update to address issues pointed by @lucasdemarchi 
Rebased on top of #29 - but without kernel related patches.

**Comment by lucasdemarchi** [comment]
Applied.

**Pull request closed without merging**